src/models/src/NITableViewModel.h

```diff
@@ -119,10 +119,10 @@ typedef enum {
  */
 @interface NITableViewModelFooter : NSObject
 
-+ (id)footerWithTitle:(NSString *)title;
-- (id)initWithTitle:(NSString *)title;
++ (nonnull id)footerWithTitle:(nonnull NSString *)title;
+- (nonnull id)initWithTitle:(nonnull NSString *)title;
 
-@property (nonatomic, copy) NSString* title;
+@property (nonatomic, copy, nonnull) NSString* title;
```

Comment:
I think this technically needs to be `nullable` because if someone calls -[NSTableViewModelFooter init] then the title will be nil 🤓

Reply:
Truth!